test/lib/ufe/testAttribute.py

```diff
@@ -1087,6 +1117,180 @@ def testObservation(self):
         self.assertEqual(ball35Obs.notifications, 3)
         self.assertEqual(globalObs.notifications, 11)
 
+    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '4024', 'Test for UFE preview version 0.4.24 and later')
+    def testObservation(self):
```

Comment:
This method has the same exact name as the one above, so I think that python will just squash the first one and only use this second one.

I don't like how there are now two test methods that need to be updated. Can you not instead just add your new code in the method above and have it execute only for ufe > 0.4.24?

Reply:
I agree with you, but tried that in a previous PR and got told to use a separate test. This created the 3 duplicate tests in testContextOps.py.

I suspect the goal is to get rid of all the intermediate tests at release time and only keep a UFE >= 4 and UFE < 4 version when releasing.